### ISSUE TITLE
ER: We need a DR for where to put the primary navigation element

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#4368]
### Emergent Requirement - Problem
We just made a decision to make the header the primary navigation element, and we might want to change that to the footer in the future.

### Issue you discovered this emergent requirement in
- #4302
- #4303

### Date discovered 
2023-04-02

### Did you have to do something temporarily
- [ ] YES
- [x] NO

### Who was involved
@roslynwythe @ExperimentsInHonesty 

### What happens if this is not addressed
We later forget all about this primary navigation tag and if we want to make some other element (like the footer be the primary) then we need to know about it.

Why we might want it to be the footer in the future:
It's possible the footer might be a better nav item for screen readers because in future its going to list all the pages

### Recommended Action Items
- [ ] Make a new issue
- [ ] Discuss with team
- [ ] Let a Team Lead know

### Potential solutions [draft]
see draft DR below https://github.com/hackforla/website/issues/4368#issuecomment-1760832253

**Comment by roslynwythe** [comment]
Draft DR: Adding the "primary" and "secondary" labels to the header and footer navigation menus

The HTML5 element `<nav>` (or alternatively the landmark role `navigation`) is used to identify a section of HTML which contains navigation links.  If there is more than one navigation landmark role or `<nav>` element in a document, each should have a unique label. This label will allow an assistive technology user to be able to quickly understand the purpose of each landmark.

A common label for the main navigation of a website is "primary" or "main".  The primary or main navigation menu is the principal navigation interface on a website. It links to the most important website pages and usually appears prominently at the top of the homepage and other pages. Primary navigation helps users to quickly find what they’re most likely to want, with minimal searching and clicks.

In the HfLA website, `<nav>` elements will be labelled as follows:
- in the site header (`_includes/main-nav.html`):
``` 
<nav class="header-nav" id="headerNav" aria-label="Primary">
```
- in the site footer (`_includes/footer.html`):
```
<nav class="footer-nav" aria-label="Secondary">
```
References:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe this looks good to me!  Whats next?  If its making an issue, go for it.

**Comment by roslynwythe** [comment]
@ExperimentsInHonesty the code changes had already been made.  A link to the above DR has been provided in the "How to Contribute to the Wiki" page so this ER is complete.
